ocs_ci/ocs/ocp.py

```diff
@@ -342,13 +345,17 @@ def wait_for_delete(self, resource_name='', timeout=60, sleep=3):
                 self.get(resource_name=resource_name)
             except CommandFailed as ex:
                 if "NotFound" in str(ex):
-                    log.info(f"{self.kind} {resource_name} got deleted successfully")
+                    log.info(
+                        f"{self.kind} {resource_name} got deleted successfully"
+                    )
                     return True
                 else:
                     raise ex
 
             if timeout < (time.time() - start_time):
-                raise TimeoutError(f"Timeout when waiting for {resource_name} to delete")
+                raise TimeoutError(
+                    f"Timeout when waiting for {resource_name} to delete"
```

Comment:
Thanks for fixing this!

Can you also call describe on the resource to get more info?
See wait_for_resource_state() under helpers.py for reference

Reply:
Added resource describe output log

Reply:
One small adjustment,
Could you move the describe output to the raise statement so it will be visible directly on the assertion when analyzing? Instead of searching it in the console log

Reply:
Done